README.md

```diff
@@ -1,2 +1,6 @@
 # Northcoders News API
 
+Installs required:
+- npm i pg - for database
+- npm i jest -D - developer dependancy - for testing
```

Comment:
this is very thorough but they can simply run the command of `npm I` and it will install everything listed in package.json!